### ISSUE TITLE
Drift between epoch seconds and "ss" seconds.

### ISSUE THREAD

**Issue by mattbailey** [moment/luxon#370]
Working on a small clock project, I noticed that seconds in epoch time is offset from (I guess, local time zone?) seconds.

ex:

```javascript
const { DateTime } = require('luxon')

setInterval(() => {
  const now = DateTime.local()
  console.log(now.toFormat('X ss'))
}, 1000 / 60)
```

If you run this (interval set for 60fps), you'll see the seconds of epoch time (`X`) change on a different interval than `ss` seconds:

```
1542407050 09
1542407050 09
1542407050 09
1542407050 09
1542407050 09
1542407050 09
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407050 10
1542407051 10
1542407051 10
1542407051 10
1542407051 10
```

Is this some obscure chronological issue, or an actual bug, or what?

Edit: same behavior in node v8 and v11, macos 10.14

**Comment by mattbailey** [comment]
This appears to be because the `X` token uses `Math.round()` rather than `Math.ceil()` or `Math.floor()`.

I'll leave the bug open because I don't know if that's intentional or not.

**Comment by icambron** [comment]
Hmm, yeah, I think it should probably be `Math.floor()`

**Comment by icambron** [comment]
Fixed in 1.7.1
